package.json

```diff
@@ -1,12 +1,27 @@
 {
-  "name": "hubba-locks",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC"
+    "name": "lockgoose",
+    "version": "0.1.0",
+    "description": "Simple locking in mongo",
+    "main": "index.js",
+    "scripts": {
+        "lint": "eslint .",
+        "test": "jest --runInBand"
+    },
+    "keywords": [
+        "locking",
+        "mongo",
+        "mongoose"
+    ],
+    "author": "Hubba",
+    "license": "MIT",
+    "peerDependencies": {
+        "mongoose": "^5.1.2"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "eslint": "~4.19.1",
+        "jest": "^22.4.4",
+        "mongodb-memory-server": "^1.7.4",
```

Comment:
ohh, this seems better than mockgoose

Reply:
found it in the jest documentation, very useful!